### ISSUE TITLE
Delete bad author names (cassette catalog ids)

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary-client#74]
See: https://github.com/internetarchive/openlibrary/issues/896 / #73 

Similar to https://github.com/internetarchive/openlibrary/issues/152 (# many Compact Cassette catalog ids have been embedded in author record name fields.
Please run this oh-so-similar check to quantify the current size of the problem:
`egrep -Ec " Cs[a-zA-Z0-9]{2,4} " ol_dump_authors_2018-02-28.txt`

cc: @leadsongdog

**Comment by LeadSongDog** [comment]
Is there a similar problem for DVDs or VHS cassettes?

**Comment by LeadSongDog** [comment]
Apparently less structured. 
https://openlibrary.org/search?q=title%3A+%22VHS%22&mode=everything gets 300 hits; most but not all are videos.
Similarly,
https://openlibrary.org/search?q=title%3A+"DVD"&mode=everything gets 1655 hits; most but not all are videos. Many are books with accompanying video, and a few are about the DVD as a subject

**Comment by hornc** [comment]
thanks @tfmorris , I am processing that list now, if I have set it up correctly, those authors should be organically removed from solr too, which will be a nice test of how the solr updater is supposed to work.

I have discovered an issue where if a deleted author is still cached in its original un-deleted state, the solr update will think it exists and not remove it :(   I think it's in https://github.com/internetarchive/openlibrary/blob/fc873f2550b3a510399a8a01de1cc428ab074b17/openlibrary/solr/data_provider.py#L170    To work around this for scripted deletes: I now delete, then immediately request the author record again and confirm it is a /type/delete , and that appears to update the cache as used by the solr updater.  I have not noticed this as a problem for works.

@mekarpeles FYI, and thanks again for those logs, they helped me track this down!
